QuadradiusR/scripts/torus.gd

```diff
@@ -139,7 +139,12 @@ func make_move(source_tile: Tile, target_tile: Tile, is_colliding: bool = false)
 		anim.play("DestroyOpponent")
 	# TODO move the sound playing to here maybe to handle dropping the piece
 	pass
-	
+
 # TODO maybe add method destroy() which would handle aftermath
 # OR destroy_piece(torus) which would make one piece destroy the other, so it could handle
 # all the consequences
+
+
+func add_power(power_id: String, power_definition_id):
+	anim.queue("CollectPowerup")  # currently this will restart color animation on every powerup pickup, it shouldnt
```

Comment:
What do you mean by `restart color animation`?

Reply:
Currently, the color tint animation starts fro 0 color to 100% color, running it again when collecting powerup will make it instantly go to 0 then animate to 100%. We don't want that when collecting additional powerups. It could be fixed by skipping the animation, but we'd lose sfx then, I need to figure out how to handle it nicely.

Reply:
>_(\ Sfx is played by orb, not torus. I added the skip mechanism